packages/@aws-cdk/aws-glue-alpha/lib/database.ts

```diff
@@ -136,3 +148,9 @@ function validateLocationUri(locationUri: string): void {
     throw new Error(`locationUri length must be (inclusively) between 1 and 1024, but was ${locationUri.length}`);
   }
 }
+
+function validateDescription(description: string): void {
+  if (description.length > 2048) {
+    throw new Error(`description length must be less than or equal to 2048, but was ${description.length}`);
```

Comment:
```suggestion
    throw new Error(`description length must be less than or equal to 2048, got ${description.length}`);
```
extremely small nit: 'got' is a bit more standardized with the design guidelines.